packages/utils/src/modules-sdk/decorators/context-parameter.ts

```diff
@@ -8,3 +8,12 @@ export function MedusaContext() {
     target.MedusaContextIndex_[propertyKey] = parameterIndex
   }
 }
+
+MedusaContext.getIndex = function (
```

Comment:
should we add the __type line 7 as well?

Reply:
Not here. That is a static property added to the class, which holds the position of the Context argument in each method.

Reply:
yes you are right my bad